packages/smithy-client/src/emitWarningIfUnsupportedVersion.ts

```diff
@@ -0,0 +1,12 @@
+export const emitWarningIfUnsupportedVersion = (version: string) => {
```

Comment:
This is not what I thought of. I thought this is going to be executed upon importing, and have side effect for bundling. How do you plan to call this function in Clients?

Reply:
> How do you plan to call this function in Clients?

We're going to call `emitWarningIfUnsupportedVersion` from `getRuntimeConfig`
PR on fork: https://github.com/trivikr/aws-sdk-js-v3/pull/156

Reply:
The PR was updated to emit warning only once.
Verified in https://github.com/trivikr/aws-sdk-js-v3/pull/156